packages/data-explorer/src/views/EntityView.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="mt-1 entity-table" v-if="activeEntityData && activeEntityData.items.length > 0 && entityMeta">
+    <div v-if="isShop && entitiesToShow.length === 0" class="alert alert-warning">
+      {{ 'dataexplorer_empty_shopping_cart' | i18n}}
+    </div>
+
+    <div class="card-columns" v-else-if="dataDisplayLayout==='cards'">
+      <entity-card
+        class="card"
+        v-for="(entity, index) in entitiesToShow"
+        :key="index"
+        :id="getEntityId(entity)"
+        :isSelected="isSelected(entity)"
+        :isShop="isShop"
+        :clickShop="toggleShoppingItems">
+
+        <!-- Card template, to be made generic-->
+        <div class="card-body">
+          <h5 class="card-title">{{entity.data.xstring}}</h5>
+          <p class="card-text">{{entity.data.xtext}}</p>
+        </div>
+
+      </entity-card>
+    </div>
+
+    <table v-else class="table">
+      <entity-table-header :header="Object.keys(entitiesToShow[0].data)" :isShop="isShop"></entity-table-header>
```

Comment:
Suggest to replace complex header from function with named computed prop

Reply:
Done. Note that this is only a temporary 'walking skeleton' table